config/runtime.exs

```diff
@@ -142,3 +142,15 @@ if config_env() in [:prod, :demo] do
   config :trento,
     api_key_authentication_enabled: System.get_env("ENABLE_API_KEY", "true") == "true"
 end
+
+secret_key_base = Application.fetch_env!(:trento, TrentoWeb.Endpoint)[:secret_key_base]
```

Comment:
Please use secret_key_base as variable, no need to refetch, you can move this config block to the same block as other runtime,

You need to put the same block in config.exs for the base config, in runtime we should only have runtime dependent config.